src/main/pg/adc.c

```diff
@@ -37,68 +37,28 @@ PG_REGISTER_WITH_RESET_FN(adcConfig_t, adcConfig, PG_ADC_CONFIG, 0);
 
 void pgResetFn_adcConfig(adcConfig_t *adcConfig)
```

Comment:
I'm not sure about this. Ok for now (with better name), but we should find something better long-term